enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -2381,7 +2381,7 @@ func (c *FirecrackerContainer) remove(ctx context.Context) error {
 		c.scratchStore = nil
 	}
 	if c.rootVBD != nil {
-		if err := c.rootVBD.Unmount(); err != nil {
+		if err := c.rootVBD.Unmount(ctx); err != nil {
```

Comment:
Discussed offline, but I think there's a chance there could be errors if we fail to unmount, and then we try to close the store while the VBD drive is still reading from it. That might be better than hanging indefinitely, but perhaps worth looking into

Reply:
Thinking about it a bit more, I think we have bigger problems if firecracker block device IO is allowed to happen after the VM is paused. Specifically, if there are COW writes in progress, it could cause filecache corruption, since we could compute the chunk digest before the in-progress write completes, then afterwards a write comes in and causes the chunk contents to change, no longer matching the digest.

I think we should treat this scenario as a separate issue, and get this PR merged to prevent user actions getting stuck.